gui/wxpython/core/testsuite/toolboxes.sh

```diff
@@ -3,5 +3,5 @@
 # run make in gui/wxpython before the test
 # run test using sh -e
 
-python $GISBASE/gui/wxpython/core/toolboxes.py doctest
-python $GISBASE/gui/wxpython/core/toolboxes.py test
+python3 $GISBASE/gui/wxpython/core/toolboxes.py doctest
+python3 $GISBASE/gui/wxpython/core/toolboxes.py test
```

Comment:
Do we have GRASS_PYTHON defined here or not? If yes, that might be the best choice, although I would prefer solution where plain python just works.